app/src/main/java/ca/marshallasch/veil/FragmentLogin.java

```diff
@@ -38,7 +42,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         //hides keyboard if white space is pressed
         layout.setOnTouchListener((view12, ev) -> {
```

Comment:
should wr link in the SO post for the reason why its okay to ignore the warning?